.github/workflows/test-service.yml

```diff
@@ -40,11 +57,14 @@ jobs:
           distribution: "adopt"
           java-version: 11
       - name: Unit tests
+        env:
+          BEACONS_CORS_ALLOWED_ORIGINS: "*"
```

Comment:
How come this was added? 👀

Reply:
Ah unit tests, shouldn't be an issue, but wondering why?

Reply:
Ah that was me trying to guess the env vars required by the unit tests. Now we're keeping application.yml in version control I can remove this